docs/src/catalyst_applications/dynamical_systems.md

```diff
@@ -0,0 +1,149 @@
+
+!!! warning
+    Both Makie and Plots.jl exports a function called `plot`. Hence, if both these packages are imported into the same session, calls to `plot` must be prepended with the package one wishes to use (e.g. `Plots.plot(sol)`).
+
+More information on how to compute basins of attractions for ODEs using DynamicalSystems.jl can be found [here](https://juliadynamics.github.io/DynamicalSystemsDocs.jl/attractors/stable/basins/).
+
+## [Computing Lyapunov exponents](@id dynamical_systems_lyapunov_exponents)
+[*Lyapunov exponents*](https://en.wikipedia.org/wiki/Lyapunov_exponent) are scalar values that can be computed for any state of an ODE. For an ODE with $n$ variables, for each state, a total of $n$ Lyapunov exponents can be computed (and these are collectively called the *Lyapunov spectrum*). Large Lyapunov exponents indicates that trajectories infinitesimally close to the given state diverge from each other. Conversely, small Lyapunov exponents suggests that trajectories converge to each others. 
+
+While Lyapunov exponents can be used for other purposes, they are primarily used to characterise [*chaotic behaviours*](https://en.wikipedia.org/wiki/Chaos_theory) (where small changes in initial conditions has large effect on the resulting trajectories). Generally, an ODE exhibit chaotic behaviours in a point in phase space if that point have *at least one* positive Lyapunov exponent. Practically, Lyapunov exponents can be computed using DynamicalSystems.jl's `lyapunovspectrum` function. Here we will use it to investigate two models, one which exhibits chaos and one which do not.
+
+First, let us consider the [Willamowski–Rössler](https://www.degruyter.com/document/doi/10.1515/zna-1980-0308/html?lang=en) model, which is known to exhibit chaotic behaviours. 
+```@example dynamical_systems_lyapunov
+using Catalyst
+wr_model = @reaction_network begin
+    k1, 2X --> 3X
+    k2, X --> 2X
+    k3, Z + 2X --> 2Z
+    k4, Y + X --> 2Y
+    k5, Y --> ∅
+    k6, 2Z --> ∅
+    k7, Z --> ∅
+end
+```
+We can simulate the model, noting that its behaviour seems chaotic.
+```@example dynamical_systems_lyapunov
+using OrdinaryDiffEq, Plots
+
+u0 = [:X => 1.5, :Y => 1.5, :Z => 1.5]
+tspan = (0.0, 100.0)
+p = [:k1 => 2.1, :k2 => 0.7, :k3 => 2.9, :k4 => 1.1, :k5 => 1.0, :k6 => 0.5, :k7 => 2.7]
+
+oprob = ODEProblem(wr_model, u0, tspan, p)
+sol = solve(oprob, Rodas5P())
+plot(sol; idxs=(:X, :Y, :Z))
+```
+Next, like when we [computed basins of attraction](@ref dynamical_systems_basins_of_attraction), we create a `CoupledODEs` corresponding to the model and state for which we wish to compute our Lyapunov spectrum (here, the `tspan` of the used `ODEProblem` have no impact on the computed Lyapunov spectrum).
+```@example dynamical_systems_lyapunov
+using DynamicalSystems
+ds = CoupledODEs(oprob, (alg = Rodas5P(autodiff=false),))
+nothing # hide
```

Comment:
`ds` prints very nicely, perhaps it is worth showing?
```suggestion
```

Reply:
Sounds good, I have added so the first one is displayed.